### DISCUSSION TITLE
Osu lazer feels smaller than Osu! (while playing

### DISCUSSION THREAD

**Discussion by Filipe402akaSimpLord** [ppy/osu#25906] [Q&A]
Maybe i'm being dumb and theres a setting to change this but i couldnt find it. 

https://youtu.be/zpgh5augP3U

**Comment by Gabixel** [comment]
You're probably looking for the "screen scaling" option. At least, that's what I'm thinking by watching the video.

![image](https://github.com/ppy/osu/assets/43073074/f5131858-6963-42d1-825f-dc4c41c9e107)

...and its related options, if you select 'gameplay' for instance:

![image](https://github.com/ppy/osu/assets/43073074/acabc75e-da84-4396-9f0f-9d45ae5d9d75)
